rebuild/src/addon.h

```diff
@@ -0,0 +1,132 @@
+#ifndef BLST_TS_ADDON_H__
+#define BLST_TS_ADDON_H__
+
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <memory>
+#include <openssl/rand.h>
+#include "napi.h"
+#include "blst.hpp"
+
+// TODO: these should come out post PR review
+using std::cout;
+using std::endl;
+
+#define BLST_TS_SECRET_KEY_LENGTH 32U
```

Comment:
I think this will create a circular dependency in the constants declaration for the addon.  I think that is what you mentioned in the next comment but let me know if you def want them moved.  I think they should stay at the top of the entrance file though to make sure circ deps don't creep in and the dependency graph is directed/non-cyclic

Reply:
because you want to build constants? can the constants not be build in individual class wrappers when `init` is being called? I am assuming it will call the constructor of these classes

Reply:
Updated.